timesketch/lib/sigma_util.py

```diff
@@ -78,19 +78,28 @@ def get_sigma_config_file(config_file=None):
     return sigma_config
 
 
-def _enrich_sigma_rule_object(rule: SigmaRule):
-    """Helper function: Returns an enriched Sigma dict given a SigmaRule.
+def _enrich_sigma_rule_object(rule: SigmaRule, parse_yaml: bool = False):
```

Comment:
This looks like the same function as in api/sigma.py? Why do we have two copies?

Reply:
https://github.com/google/timesketch/issues/2633